<?xml version="1.0"?>
<launch>
  <node pkg="camera_apps" type="object_detector_node" name="object_detector_node"
      output="screen" >
    <param name="model_path" value="$(find camera_apps)/model" />
  </node>    

  <node pkg="camera_apps" type="object_state_estimator_node" name="object_state_estimator_node"
      output="screen">
      <param name="mean_k" type="int" value="10"/>
      <param name="std_dev_th" type="double" value="0.2"/>
      <param name="leafsize" type="double" value="0.05"/>
      <param name="points_limit" type="int" value="10000"/>
  </node>    

  <node pkg="tf2_ros" type="static_transform_publisher" name="tf_fixed_to_origin"
      args="0 0 0 4.7124 0 4.7124 camera_fixed_frame camera_color_optical_frame">
  </node>
  <node pkg="rviz" name="rviz" type="rviz" args="-d $(find camera_apps)/config/rviz/camera_apps.rviz" output="screen"/>

  <node name="image_republish" pkg="image_transport" type="republish" args="compressed raw">
      <remap from="in" to="/camera/color/image_raw/" />
      <remap from="out" to="/camera/color/image_raw" />
  </node>
</launch>
